src/generators/default_map_generator_job.cpp

```diff
@@ -19,26 +19,229 @@
 
 #include "global.hpp"
 
+#include "default_map_generator_job.hpp"
+#include "formula/string_utils.hpp"
+#include "game_config_manager.hpp"
 #include "gettext.hpp"
 #include "language.hpp"
 #include "log.hpp"
 #include "map/map.hpp"
-#include "map_generator.hpp"// mapgen_esxeption
-#include "default_map_generator_job.hpp"
+#include "map_generator.hpp" // mapgen_esxeption
 #include "pathfind/pathfind.hpp"
 #include "pathutils.hpp"
 #include "util.hpp"
-#include "wml_exception.hpp"
-#include "formula/string_utils.hpp"
 #include "utils/name_generator_factory.hpp"
+#include "seed_rng.hpp"
+#include "wml_exception.hpp"
+
 #include <SDL.h>
-#include "game_config_manager.hpp"
 
-#include "seed_rng.hpp"
 static lg::log_domain log_mapgen("mapgen");
 #define ERR_NG LOG_STREAM(err, log_mapgen)
 #define LOG_NG LOG_STREAM(info, log_mapgen)
 
+
+typedef std::vector<std::vector<int> > height_map;
+typedef t_translation::t_map terrain_map;
+typedef map_location location;
+
+namespace {
```

Comment:
Did you make any changes in this section of code, or just move it up here without change?